lessons/09-clientside-frameworks/README.md

```diff
@@ -1,33 +1,33 @@
 #Lesson 9 - Client-side Javascript Frameworks
 
-##Intro
+##Introduction
 So far we have seen how to organize an increasingly complex system of components on our node server. 
-We will now take a side step to discuss another organizational system in which instead of having lot of structure on the server, we have that structure on the client. 
+We will now take a side step to discuss another organizational system in which instead of having lot of structure on the **server**, we have that structure on the **client**. 
 
-##Basic Idea
+##Current Idea
 In our traditional system, with most clicks you make a call to your server for an html page.
 You have seen how this can work with templating by rendering html on the server and then sending it across to the client as just html. 
 The image below shows this traditional system: 
-![serverOrganizedSystem](images/traditionalWebCommunication.png)
+![serverOrganizedSystem](images/traditionalWebCommunicationNew.png)
```

Comment:
If you're going to replace the images, you may want to just replace them so that we don't have old, unused image assets hanging around.